src/EditorFeatures/CSharpTest/GenerateFromMembers/AddConstructorParametersFromMembers/AddConstructorParametersFromMembersTests.cs

```diff
@@ -167,8 +177,7 @@ public Program(int i, string s, bool b = false) : this(i)
         this.s = s;
         this.b = b;
     }
-}",
-index: 1);
+}", index: 3, title: string.Format(FeaturesResources.Add_to_0, "Program(int, string)"));
```

Comment:
❓ Why is this not `Add_optional_parameters_to_0`?

Reply:
➡️ The 'optional' indication is part of the parent action. This seems a reasonable limitation in tests with the current approach being a substantial improvement over what we were able to test previously.